Neutron.h

```diff
@@ -0,0 +1,61 @@
+/* 
+ @file      Neutron.h
+ @brief     contains Neutron class
+ @author    Luke Eure
+ @date      January 8 2016
+*/
+
+#ifndef NEUTRON_H
+#define NEUTRON_H
+
+#include <vector>
+#include <cmath>
+#include <stdio.h>
+#include <math.h>
+#include <iostream>
+
+#include "Distributions.h"
+
+class Neutron {
```

Comment:
Nice!  This is something I've been doing lately (having a class for the particle)... I think it makes everything better :-)